actors/miner/src/lib.rs

```diff
@@ -125,13 +125,22 @@ pub enum Method {
     GetBeneficiary = 31,
     ExtendSectorExpiration2 = 32,
     // Method numbers derived from FRC-0042 standards
+    ChangeWorkerAddressExported = frc42_dispatch::method_hash!("ChangeWorkerAddress"),
+    ChangePeerIDExported = frc42_dispatch::method_hash!("ChangePeerID"),
+    WithdrawBalanceExported = frc42_dispatch::method_hash!("WithdrawBalance"),
+    ChangeMultiaddrsExported = frc42_dispatch::method_hash!("ChangeMultiaddrs"),
+    ConfirmUpdateWorkerKeyExported = frc42_dispatch::method_hash!("ConfirmUpdateWorkerKey"),
```

Comment:
Can fix this poor name here? This should be `ConfirmChangeWorkerAddress` because it confirms a `ChangeWorkerAddress`. Prior to this, it's just a symbol that we could change for a fixed underlying method number, but this would lock it in.

Reply:
Yeah, definitely.